internal/system/agent/utils.go

```diff
@@ -29,3 +34,65 @@ func pingHandler(w http.ResponseWriter, r *http.Request) {
 		LoggingClient.Error("Error writing pong: " + err.Error())
 	}
 }
+
+const (
+	manifestDirectory = "./res"
+	manifestDefault   = "manifest.toml"
```

Comment:
As I said above, consider eliminating manifest.toml and folding that information into configuration.toml. If you do that, you don't need this go file with all of its duplicated logic around finding paths, etc.